components/Pages/CarePackages/ReviewPackageDetails/index.js

```diff
@@ -14,85 +14,106 @@ const links = [
   { text: 'Care Package', href: '#care-package' },
   { text: 'Weekly Additional Need', href: '#weekly-additional-need' },
   { text: 'One Off Additional Need', href: '#on-off-additional-need' },
+  { text: 'Funded Nursing Care', href: '#funded-nursing-care' },
   { text: 'Care charges', href: '#care-charges' },
   { text: 'Summary', href: '#summary' },
 ];
 
-export const ReviewPackageDetails = ({ userDetails, packageInfoItems = [], summary = [] }) => {
-  const router = useRouter();
-  const packageId = router.query.guid;
+const breadcrumbs = [
+  { text: 'Home', href:  '/' },
+  { text: 'Broker Portal', href: BROKER_PORTAL_ROUTE },
+  { text: 'Full overview' },
+];
 
+export const ReviewPackageDetails = ({
+  userDetails,
+  packageId,
+  packageInfoItems = [],
+  summary = [],
+  supplierName,
+  title = 'Nursing Care',
+  subTitle = 'Package details',
+}) => {
+  const router = useRouter();
   const [isOpenedPopup, setIsOpenedPopup] = useState(false);
 
+  const goBack = () => router.back();
+
   const redirectToBrokerPackage = () => {
-    router.push(getBrokerPackageRoute(packageId));
+    router.push({
+      pathname: getBrokerPackageRoute(packageId),
+      query: { supplierName },
```

Comment:
You can remove `query`. Broker package page was reworked. It doesn't need `supplierName` anymore